checker/exported_ident_doc_checker_test.go

```diff
@@ -168,3 +168,139 @@ func TestExportedIdentDocChecker(t *testing.T) {
 		})
 	}
 }
+
+func TestExportedIdentDocCheckerConfig(t *testing.T) {
+	tests := []struct {
+		description string
+		config      *ExportedIdentDocCheckerConfig
```

Comment:
Why not add a `config` to the top test and reuse it? I think there's no reason to separate those tests - it's just testing different input against different configuration.

Reply:
Done.